emf/loadflow_tool/helper.py

```diff
@@ -153,15 +154,24 @@ def load_model(opdm_objects: List[dict]):
     model_data["network_valid"] = network.validate().name
 
     # Network model import reporter data
-    # model_data["import_report"] = json.loads(import_report.to_json())
+    model_data["import_report"] = json.loads(import_report.to_json())
     # model_data["import_report_str"] = str(import_report)
 
     return model_data
 
 
 def opdmprofile_to_bytes(opdm_profile):
+    # Temporary fix: input data (['opdm:Profile']['DATA']) can be a zip file, figure it out and extract
+    # before proceeding further
     data = BytesIO(opdm_profile['opdm:Profile']['DATA'])
-    data.name = opdm_profile['opdm:Profile']['pmd:fileName']
+    file_name = opdm_profile['opdm:Profile']['pmd:fileName']
+    if zipfile.is_zipfile(data) and not file_name.endswith('.zip'):
```

Comment:
This check and workaround should not be handled here

Reply:
Removed in the branch mr-66-merge-for-RMM